block support is not working yet, and including it would make netdde have to
spuriously statically link against libdde_linux26_block, so let's not link it in
until we have libdde working properly as a shared library.

Index: hurd-debian/libmachdev/Makefile
===================================================================
--- hurd-debian.orig/libmachdev/Makefile
+++ hurd-debian/libmachdev/Makefile
@@ -20,12 +20,12 @@ makemode := library
 libname = libmachdev
 
 SRCS = deviceUser.c machUser.c net.c ds_routines.c queue.c trivfs_server.c \
-       device_replyUser.c deviceServer.c notifyServer.c misc.c block.c
+       device_replyUser.c deviceServer.c notifyServer.c misc.c
 LCLHDRS = dev_hdr.h device_emul.h ds_routines.h vm_param.h \
 	  util.h queue.h io_req.h if_ether.h machdev.h linux-errno.h \
 	  errno-base.h
 installhdrs = machdev.h
-HURDLIBS = ports trivfs ddekit bpf
+HURDLIBS = ports trivfs bpf
 OTHERLIBS = -lpthread
 OBJS = $(SRCS:.c=.o) $(MIGSTUBS)
 MIGSFLAGS = -imacros $(srcdir)/mig-mutate.h
